index.js

```diff
@@ -308,15 +308,17 @@ async function ledgerSignTransaction(seriesCb, transaction, account, callback) {
 }
 
 ledgerWorker.on('message', function (message) {
-  if (message.connected && network && (!ledgerComm || !ledgerAccounts.length)) {
+  if (message.connected && (!ledgerComm || !ledgerAccounts.length)) {
```

Comment:
I don't think this works. I get the following output (because it's looping to get ledger accounts, or it's not connected to a network when trying to do `populateLedgerAccounts`:

![image](https://user-images.githubusercontent.com/8069294/30831819-ce23acd6-a240-11e7-8122-03989e9930bd.png)

The above stops once I connect to a node (`connect node 1.2.3.4`)

Reply:
yes "last time code change not tested before pushing" error 😥

Reply:
We're all guilty of it 😁 do you want me to fix? I only added this network check in but can re-test the whole process later if you want me to?